spec/features/shared_examples/sequencing.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+shared_examples 'a sequencing procedure' do
+  it 'can be processed', warren: true do
+    login_user(user)
+    visit pipeline_path(pipeline)
+    within('#available-requests') { all('input[type=checkbox]', count: 2).each(&:check) }
+    first(:button, 'Submit').click
+    click_on('Specify Dilution Volume')
+
+    all(:field, 'Concentration').each_with_index { |field, index| field.fill_in(with: 1.2 + index) }
+
+    click_on 'Next step'
+
+    fill_in('PhiX Barcode', with: 'Not a barcode')
+
+    click_on 'Next step'
+
+    expect(page).to have_content("Can't find a spiked hybridization buffer with barcode Not a barcode")
+
+    find('#sample-2-checkbox').uncheck
+
+    fill_in('PhiX Barcode', with: spiked_buffer.machine_barcode)
+
+    click_on 'Next step'
+
+    find('#sample-1-checkbox').uncheck
+
+    expect(page).to have_content('Request 1 :')
+
+    click_on 'Next step'
+
+    find('#sample-1-checkbox').uncheck
+
+    select('XP', from: 'Workflow (Standard or Xp)')
+    fill_in('Lane loading concentration (pM)', with: 23)
+    fill_in('+4 field of weirdness', with: 'Check stored')
+
+    click_on 'Next step'
+
+    find('#sample-2-checkbox').uncheck
+
+    # Pending question on issue#3225 may be populated with previous v
```

Comment:
is this meant to say 'with previous value'?

Reply:
assume so! will update